src/Data/Fin/Properties.agda

```diff
@@ -145,6 +152,24 @@ fromℕ-def : ∀ n → fromℕ n ≡ fromℕ< ℕₚ.≤-refl
 fromℕ-def zero    = refl
 fromℕ-def (suc n) = cong suc (fromℕ-def n)
 
+fromℕ<-irrelevant : ∀ m n {o} → m ≡ n
+                  → (m<o : m ℕ.< o)
```

Comment:
Arrows should go at the end of the lines rather than the beginning of lines as per the style guide (here and below)